spec/spec_helper.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+    config.order = :random
```

Comment:
Order is random by default I think.

Reply:
Defaults one is `defined` which sort of means, 'in a row'.
https://relishapp.com/rspec/rspec-core/docs/command-line/order

Reply:
oh, np